src/EnergyPlus/CMakeLists.txt

```diff
@@ -700,8 +699,26 @@ set(INPUTPARSING_SRC
 
 create_src_groups("${INPUTPARSING_SRC}")
 
-add_custom_target(
-  GenerateEmbeddedEpJSONSchema
+configure_file("${PROJECT_SOURCE_DIR}/idd/Energy+.idd.in" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Energy+.idd")
+
+add_custom_command(
+        OUTPUT "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Energy+.schema.epJSON"
+        COMMAND ${Python_EXECUTABLE} "${PROJECT_SOURCE_DIR}/scripts/dev/generate_epJSON_schema/generate_epJSON_schema.py" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}"
+        DEPENDS
+        "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Energy+.idd"
+        "${PROJECT_SOURCE_DIR}/scripts/dev/generate_epJSON_schema/generate_epJSON_schema.py"
+        "${PROJECT_SOURCE_DIR}/scripts/dev/generate_epJSON_schema/idd_parser.py"
+        "${PROJECT_SOURCE_DIR}/scripts/dev/generate_epJSON_schema/modify_schema.py"
+)
+
+# Since multiple targets depend on the JSON Schema, we need to make a custom target for it so the dependent targets don't try to run the custom command at the same time.
+# (see: https://cmake.org/cmake/help/latest/command/add_custom_command.html?highlight=add_custom_command#example-generating-files-for-multiple-targets)
+
+add_custom_target(epjson_schema ALL DEPENDS "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Energy+.schema.epJSON")
```

Comment:
This target needs to be added [to the `generate_embeddable_epJSON_schema` CMakeLists](https://github.com/NREL/EnergyPlus/blob/10053-epJSON-build-dependencies/scripts/dev/generate_embeddable_epJSON_schema/CMakeLists.txt#L3-L4).

` add_dependencies(generate_embeddable_epJSON_schema epjson_schema)`

`generate_embeddable_epJSON_schema` depends on `Energy+.schema.epJSON` as an input.

Reply:
Yeah, you are correct. `Energy+.schema.epJSON` needs to be passed into that executable but isn't needed to compile that executable.